libs/design-system/src/atoms/buttons/buttons.tsx

```diff
@@ -0,0 +1,64 @@
+import { ReactNode } from 'react'; // createElement,
+
+import './../../index.css';
+
+import { cn } from '../../shared/classnames';
+
+interface ButtonProps {
+  /**
+   * Children elements
+   */
+  children?: ReactNode;
+
+  /**
+   * The className to apply to the text. This will override any built-in style.
+   *
+   * this is useful for layout positioning, responsive sizing, and for any stylistic
+   * exception that is not part of our design system (eg. a specific size used only once)
+   */
+  className?: string;
+
+  /**
+   * The button type to use (default: `primary`)
+   */
+  type?: 'primary' | 'secondary';
+
+  /**
+   * A callback to detect clicks on the text (default: undefined)
+   */
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+const fontSizeOptions = {
+  primary: 'text-5',
+  secondary: 'text-3',
+};
+
+const borderRadiusOptions = {
+  primary: 'rounded-md',
+  secondary: 'rounded',
+};
+
+const paddingOptions = {
+  primary: 'px-4 py-2',
+  secondary: 'px-2 py-1',
+};
```

Comment:
Maybe slightly overengineered especially if we end up with 3 or 4 button types.

Something like:
```
const typeSpecificClasses = {
    primary: 'px-4 py-2 rounded-md text-5',
    secondary: '...'
}


[...]

      className={cn(
        'text-white bg-ocean-blue shadow-lg shadow-grey-light hover:bg-green transition-colors duration-300 ease-in-out',
        typeSpecificClasses[type],
        className
      )}
```

could be enough. What do you think?

Reply:
I agree, let's go

Reply:
Yes. As I went over the code again I finally thought it made more sense like this.